src/components/Stepper/src/Stepper.vue

```diff
@@ -13,9 +13,33 @@
 		>
 			<minus :class="$s.Icon" />
 		</m-button>
-		<span :class="$s.Quantity">
-			{{ value }}
-		</span>
+		<div
+			:class="$s.Quantity"
+		>
+			<input
+				v-if="isSettingManualValue"
+				ref="manualInput"
+				v-model="manualValue"
+				:class="$s.QuantityManualInput"
+				:min="min"
+				:max="max"
+				type="number"
+				inputmode="numeric"
+				align="center"
+				@blur="commitManualValue"
+				@keyup.enter="commitManualValue"
+			>
+			<span
```

Comment:
instead of having `<input v-if="editing"><span v-else>` you can have `<input :disabled="!editing">` and then you don't need the `span` (since you already styled the `input` to look like a `span` anyway). i think this should work, if not lemme know.

Reply:
I agree - do we need to have a separate span? Seems like it'd be simpler to just have the input.

Reply:
Yeah you're right, I definitely can!

Reply:
Actually, scratch that. I was using the input to auto-resize the input based on the value.

Reply:
We're suggesting to get rid of the `span` and leave only the `input`, is that not possible? 🤔

Reply:
To clarify, the reason I've left this is to allow the input to be resized.
Basically, the `span` will adjust to the width of the entered content as the user updates the value. The `input` width is tied to the containers width, and so it will resize with those changes.
This seemed like the least hacky solution to get the input to automatically resize itself, but I'm open to other suggestions to get the same effect. Otherwise, the input will have to be a set width, which might result in a jankier experience.